sdk/communication/tests.yml

```diff
@@ -4,6 +4,5 @@ extends:
   template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
   parameters:
     ServiceDirectory: communication
-
-    # Remove the SubscriptionConfiguration to test agains the public cloud
-    SubscriptionConfiguration: $(sub-config-azure-cloud-dogfood-test-resources)
\ No newline at end of file
+    EnvVars:
+      AZURE_COMMUNICATION_TEST_MODE: Record
```

Comment:
Does this mean we will run in 'Record' mode on CI ? My understanding is that it should run on Playback mode 
@JoshLove-msft  is that right ?

Reply:
I had a different understanding of Recorded vs Live tests, but I may be wrong.
I will talk to the azure folks today to clarify how this should work.